bundles/rocky9.1/bundle.sh.gotmpl

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+yum -y install epel-release gettext yum-utils createrepo dnf-utils
+yum clean all
+TMP_DIR="$(mktemp -d repodata-XXXX)"
+cp packages.txt "${TMP_DIR}"
+pushd "${TMP_DIR}"
```

Comment:
Instead of writing any of these files to a temp directory, what do you think about placing everything that is being used for a run into a timestamped directory so that users can fully see what was used when running the container?